integration/testdata/custom-test/skaffold.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: skaffold/v2beta13
+kind: Config
+build:
+  artifacts:
+  - image: custom-test-example
+test:
+  - image: custom-test-example
+    custom:
+      - command: echo "bar" > foo
+        dependencies:
+          paths:
+          -  'foo*' 
+deploy:
+ kubectl:
+   manifests:
+   - pod.yaml
```

Comment:
this test project runs into an infinite loop of rebuilds. You should try running `skaffold dev` in this directory to understand what I'm saying. 

I think the approach is good, but instead of having `foo` file be a build dependency and also modifying it in the test command, have another `test` file and the custom test command writes to that `test` file which has nothing to do with the build stage.

Reply:
Updated the docker file to remove the build dependency.